app/models/comment.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  belongs_to :user, optional: true
+  belongs_to :policy
```

Comment:
I can imagine a future where we switch this for a polymorphic association, given that lots of things could need comments.